src/video.tsx

```diff
@@ -15,6 +9,7 @@ export type VideoError = MediaError | null;
 export interface VideoState {
   status: VideoStatus;
   currentTime: number;
+  currentActiveCues: (kind: VideoTextTrackKind, lang: string) => TextTrackCueList | null | undefined;
```

Comment:
I think this goes against the current design, up to this point everything under `videoState` was plain properties and now we got a method. 

Isn't it the `actions` a more suitable place for this?

Reply:
My understanding is that `VideoState` would be "read-only" while `VideoActions` is for mutating the state?

Here I'm just looking at reading into `HTMLMediaElement.textTracks?`, the only problem is that that attribute returns a `TextTrackList` that is a bit uneasy for the user: 
- it's a `Iterable` but not an `Array` (calling forEach/filter/map on this is a pain),
- the only method that seems user-friendly is `getTrackById()`, but that imply users control the "id" they pass to it, which is the same "id" property on the <track /> HTML element (wich user don't control)

So basically my idea was to construct this "id" with (kind, lang), and ask users to pass these two params that they own, so that I can return them the proper entry from the `TextTrackList`.

There's a nifty feature of `getTrackById()` that I'm using: if there's several tracks with the same "id" (illegal, but may happen), that function only returns the first entry.

Reply:
Right! 

Yeah, you approach makes sense now. We should use actions only to mutate the state and this is purely for reading